cmd/storagenode/main.go

```diff
@@ -315,6 +385,11 @@ func isOperatorWalletValid(wallet string) error {
 	return nil
 }
 
+// clr uses ANSI escape codes to clear the screen
+func clr() {
+	print("\033[H\033[2J")
```

Comment:
for windows you need to invoke cls https://stackoverflow.com/questions/22891644/how-can-i-clear-the-terminal-screen-in-go